rust/Cargo.lock

```diff
@@ -48,7 +48,7 @@ version = "0.7.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "fcb51a0695d8f838b1ee009b3fbf66bda078cd64590202a864a8f3e8c4315c47"
 dependencies = [
- "getrandom 0.2.6",
```

Comment:
@Stebalien this fine?

Reply:
Yeah. Not sure why it's happening, but this should be safe.